bin/calyptos

```diff
@@ -3,8 +3,28 @@ import argparse
 from fabric.colors import yellow
 from stevedore import driver, extension
 from calyptos.rolebuilder import RoleBuilder
+import os
+import shutil
+
+dot_dir = os.path.expanduser('~/.calyptos')
+default_cfgfile = os.path.join(dot_dir + "/" + "config.yml")
```

Comment:
when using os.path.join its intended to be used as follows:

```default_cfgfile = os.path.join(dot_dir, "config.yml")```

Its functionality is to join add those /'s for you intelligently (ie removing duplicate /'s and such)